en/docs/learn/configuring-password-patterns.md

```diff
@@ -0,0 +1,92 @@
+# Configuring Password Patterns
+
+WSO2 Identity Server enables defining custom [password policies](../../learn/configuring-password-policies) and enforce them at the point of user creation. This enables complying with password policies deem necessary through various regulations. WSO2 Identity Server enables defining custom [password policies](../../learn/configuring-password-policies) and enforce them at the point of user creation. This enables complying with password policies deem necessary through various regulations. 
```

Comment:
and enforcing them

Reply:
This whole thing has been repeated somehow. 
"WSO2 Identity Server enables defining custom [password policies](../../learn/configuring-password-policies) and enforce them at the point of user creation. This enables complying with password policies deem necessary through various regulations. "